tests/Bedrock.Framework.Experimental.Tests/Http1Tests.cs

```diff
@@ -0,0 +1,48 @@
+using System.Buffers;
+using System.Net.Http;
+using System.Text;
+using Bedrock.Framework.Protocols;
+using Xunit;
+
+namespace Bedrock.Framework.Experimental.Tests
```

Comment:
I think @davidfowl just removed the Bedrock.Framework.Experimental.Tests folder, I'd move this into Bedrock.Framework.Tests: https://github.com/davidfowl/BedrockFramework/tree/master/tests/Bedrock.Framework.Tests